zerver/lib/webhooks/git.py

```diff
@@ -166,6 +166,7 @@ def get_pull_request_event_message(
     user_name: str,
     action: str,
     url: str,
+    *,
```

Comment:
Can these go all the way at the beginning of the parameter list? That way we ensure that call sites continue to pass all the arguments by name, so we lock in place the improvement you've made.

Reply:
Sure, updated the PR to do so.